packages/ui/tailwind.config.js

```diff
@@ -1,6 +1,6 @@
 const config = {
   content: ["./src/**/*.tsx"],
-  plugins: [require("@tailwindcss/forms")],
+  plugins: [require("@tailwindcss/forms")]
```

Comment:
Are your Prettier and VS Code set up properly? Or: do you open the entire monorepo as a workspace, or subfolders?

We have `trailingComma: "es5"` in our root `prettier.config.js`, which should force the trailing comma.